clients/venice-admin-tool/src/test/java/com/linkedin/venice/venice_paths.txt

```diff
@@ -0,0 +1,21 @@
+/venice-parent/cluster2
```

Comment:
This file is missing `/venice-parent`. Can you verify the algorithm? Also, can we also please sort the lines in the output?